### PR TITLE
Additional storage options

### PR DESCRIPTION
**Pull request by katzmo**
* Choose between sessionStorage, localStorage or cookies
* Set expiration for cookies

Depends on https://github.com/moreonion/jquery.formprefill/pull/2

### CODE REVIEW

File: webform_prefill.module [hunk 1]
```diff
@@ -44,15 +48,16 @@ function webform_prefill_libraries_info() {
 }
 
 /**
- * Implements hook_form_FORM_ID_alter().
```
Comment (torotil) [inline webform_prefill.module:47]: Coding-style would be `Implements hook_form_FORM_ID_alter() for webform_client_form().`

**Review by torotil — CHANGES_REQUESTED** [review]
… and please allow for multiple enabled storage backends (as does the jQuery plugin).

**Review by torotil — APPROVED** [review]

**Comment by torotil** [comment]
Have you tested that this still works after you’ve modified the value in the variables interface?

**Comment by katzmo** [comment]
Now the `webform_prefill_storage` variable should work with strings and arrays and variable_admin.

**Pull request merged**